scripts/lint.js

```diff
@@ -8,20 +8,30 @@ process.chdir(base)
 const reportOnly = argv.reportOnly
 
 ;(async () => {
-    !argv.noEslint
-        ? await spawn('eslint', [
-              '--ignore-path',
-              '.prettierignore',
-              '--ext',
-              'tsx,ts,jsx,js',
-              ...(argv._.length ? argv._ : ['./src/']),
-              ...(reportOnly ? [] : ['--cache', '--fix']),
-          ])
-        : null
-    !argv.noPrettier
-        ? await spawn('prettier', [
-              ...(argv._.length ? argv._ : ['./**/*.{ts,tsx,jsx,js}']),
-              ...(reportOnly ? ['--check', '--loglevel', 'log'] : ['--write', '--loglevel', 'warn']),
-          ])
-        : null
+    if (!argv.noEslint)
+        await spawn(
+            'node',
+            [
+                // Guess what happens if you don't do this?
+                // Path resolution makes a \n, which becomes...
+                path.join(base, 'node_modules/eslint/bin/eslint.js'),
```

Comment:
This could be a problem in case of using pnpm.

Reply:
What about require.resolve('eslint') or something?

Reply:
pnpm will not break because both are explicit dependencies.

Reply:
doing a resolve sounds good but is pointless -- they are never going to resolve into global deps or the installation is broken. and if they are not global does they are going to be exactly there because they are explicitly listed as deps.

Reply:
reasonable. but hard code a path is little hack with me

Reply:
aghflaljdgajdlaj